packages/design-system/src/components/Dialog/Dialog.tsx

```diff
@@ -177,20 +151,16 @@ export const Dialog = (props: DialogProps) => {
       >
         <header className={headerClassNames}>
           {heading && (
-            <h1 className="ds-h2" id={headingId} ref={headingRef}>
+            <h1 className="ds-c-dialog__heading" id={headingId} ref={headingRef}>
```

Comment:
🤔 does this h1 need to be evaluated? wondering if this component should be able to change heading levels, or if because it's a dialog it doesn't have to follow the 1 h1 per page rule 

cc: @SparrowSpit

Reply:
Given flexibility that this component must have combined with the unreliability of devs to choose the right heading, I vote for now to leave as is. We can revisit later if the there is enough steam to do so. Good question though.